core/ui/EditTemplate.tid

```diff
@@ -1,21 +1,22 @@
 title: $:/core/ui/EditTemplate
 
-\define actions()
-<$action-sendmessage $message="tm-add-tag" $param={{$:/temp/NewTagName}}/>
-<$action-deletetiddler $tiddler="$:/temp/NewTagName"/>
-<$action-sendmessage $message="tm-add-field" $name={{$:/temp/newfieldname}} $value={{$:/temp/newfieldvalue}}/>
-<$action-deletetiddler $tiddler="$:/temp/newfieldname"/>
-<$action-deletetiddler $tiddler="$:/temp/newfieldvalue"/>
+\define save-tiddler-actions()
+<$action-sendmessage $message="tm-add-tag" $param={{$(newTagNameTiddler)$}}/>
```

Comment:
Is there any hope of getting rid of the `$()$` constructions here?

Reply:
@Jermolene - perhaps by `$param={{ [<newTagNameTiddler>get[text] }}}` ?

Reply:
I think so

Reply:
I think it's resolved now